testsuite/integration-arquillian/tests/base/src/test/resources/arquillian.xml

```diff
@@ -36,13 +36,19 @@
         <property name="firefoxBinary">${firefox_binary}</property> <!-- we need to use 'firefoxBinary' instead of 'firefox_binary' due to some weird conflict with Appium -->
         <property name="firefoxLogLevel">OFF</property>
         <property name="firefoxLegacy">${firefoxLegacyDriver}</property>
+        <property name="firefoxDriverBinary">${firefoxDriverBinary}</property>
+        <property name="firefoxDriverVersion">${firefoxDriverVersion}</property>
```

Comment:
Probably that's a silly question, but shouldn't we also have a IE Driver version here?